src/components/base/modal/CommonModal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Modal, TouchableOpacity, View } from 'react-native';
+import styled from 'styled-components/native';
+import size from '../../../utils/size';
+
+const CommonModal = ({
+  showModal,
+  setShowModal,
```

Comment:
```suggestion
  onClose,
```

여기서 setShowModal은 close할때만 사용하고 있으니, setState보다는 함수로 하는 것이 더 좋을 것 같습니다.
예) `onRequestClose={onClose}`, `onPressOut={onClose}`